tutorial/lesson-1/MORE.md

```diff
@@ -282,14 +282,20 @@ to different URL paths or routes.
 ## Index.html
 
 Since this is your blog a more customized front page is probably preferred. So
-far there is no `site` directory and no `index.html` file. Those file are
-virtual or taken from the gem `export` directory. The default files for a site
-can be generated by running the `wabur init` command with a `--site` option.
+far there is no `site` directory or an `index.html` file. Those files are
+virtual, by default. In other words, they're accessed from within the `wabur`
+gem itself.
+
+To customize these files easily, they can be generated at the current WAB
+workspace by running the `wabur init` command with a `--site` option.
 
 ```
 $ wabur init --site
 ```
 
+> Pro-tip: Alternatively, the `--site` option can be passed to `wabur new` as
+well, to generate these files alongwith the config files and UI Controller.
```

Comment:
spelling: 'alongwith' should be 'along with'. Same in the MORE.md file.

Reply:
ah yes.. my bad.. :grinning: